NitroxServer/GameLogic/PlayerManager.cs

```diff
@@ -135,6 +140,12 @@ public void PlayerDisconnected(NitroxConnection connection)
                 }
 
                 assetsByConnection.Remove(connection);
+
+                if (ConnectedPlayers().Count == 0)
+                {
+                    Server.Instance.Save();
+                    Server.Instance.DisablePeriodicSaving();
```

Comment:
Probably should swap these so that the timer is stopped before saving in case the timer triggers at the same time as this save call.